doc/api/_api_api.rst

```diff
@@ -0,0 +1,13 @@
+*******************
```

Comment:
I'm confused why we are making a documentation page for something we are making private?

Reply:
This targets contributors. It may be easier to read this in HTML than in plaintext.

Pandas does something similar. See the warning note at https://pandas.pydata.org/docs/reference/index.html?highlight=private.

Reply:
Do we need similar warnings at the beginning of `_api.deprecation`?

Reply:
Yes makes sense. I'll also extend the warning to explicitly mention that these Functions are only for developers/contributors.

Reply:
We read the docs too!

Reply:
Sigh, had a stale browser and only saw the first comment when I posted, sorry for the noise.